src/ccl_power.c

```diff
@@ -1638,30 +1638,55 @@ typedef struct {
   int* status;
 } SigmaR_pars;
 
-static double sigmaR_integrand(double lk,void *params)
+typedef struct {
+  ccl_cosmology *cosmo;
+  double R;
+  int* status;
+} SigmaV_pars;
+
+static double w_tophat(double kR)
 {
-  SigmaR_pars *par=(SigmaR_pars *)params;
-  
-  double k=pow(10.,lk);
-  double pk=ccl_linear_matter_power(par->cosmo,k, 1.,par->status);
-  double kR=k*par->R;
   double w;
+
   if(kR<0.1) {
     w =1.-0.1*kR*kR+0.003571429*kR*kR*kR*kR
```

Comment:
Would be good to document where these magic numbers come from.

Reply:
Done